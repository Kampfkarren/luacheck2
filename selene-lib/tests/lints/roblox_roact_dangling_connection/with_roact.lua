local Roact

local function Component()
    Roact.useEffect(function()
        a:Connect()
    end)
end
