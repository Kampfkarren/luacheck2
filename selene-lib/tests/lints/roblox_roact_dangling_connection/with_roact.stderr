error[roblox_roact_dangling_connection]: disconnect the connection in the useEffect cleanup function
  ┌─ with_roact.lua:5:9
  │
5 │         a:Connect()
  │         ^^^^^^^^^^^

